node/sequelize/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "sequelize",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "dependencies": {
+    "body-parser": "~1.15.2",
+    "cookie-parser": "~1.4.3",
+    "debug": "~2.2.0",
+    "express": "~4.14.0",
+    "jade": "~1.11.0",
+    "morgan": "~1.7.0",
+    "sequelize-cli": "^2.4.0",
```

Comment:
You also need a dependency for `sequelize` (the ORM, not this package) here. `npm start` didn't work for me until I added:

```
    "sequelize": "^3.23.6",
```